lncfg/db.go

```diff
@@ -392,6 +416,95 @@ func (db *DB) GetBackends(ctx context.Context, chanDBPath,
 			Remote:     true,
 			CloseFuncs: closeFuncs,
 		}, nil
+
+	case SqliteBackend:
+		// Note that for sqlite, we put kv tables for the channel.db,
+		// wtclient.db, macaroon.db and sphinxreplay.db all in the
+		// lnd.db sqlite file. The only db kept separate is wallet.db.
+		// This is due to the fact that sqlite only supports one writer
+		// at a time which would cause deadlocks in the code due to the
+		// wallet db often being accessed during a write to another db.
+		sqliteBackend, err := kvdb.Open(
```

Comment:
If we do `chanDBPath` here, then it'll live in the `graph` directory which is closer to the layout rn.